charts/generic-prometheus-alerts/templates/ingress-alerts.yaml

```diff
@@ -28,6 +28,17 @@ spec:
           for: 1m
           labels:
             severity: {{ .Values.alertSeverity }}
+        - alert: High503ErrorRateOnHealthEndpoint
+          annotations:
+            message: High rate of 503 errors detected on /health path in Ingress {{`{{`}} $labels.exported_namespace {{`}}`}}/{{`{{`}} $labels.ingress {{`}}`}}.
+            runbook_url: {{ .Values.runbookUrl }}ingress-503-error-responses
+            dashboard_url: {{ $.Values.grafanaUrl }}/d/golden-signals/golden-signals?orgId=1&var-namespace={{ $targetNamespace }}&var-service={{ $targetApplication }}
+          expr: |-
+            avg(rate(nginx_ingress_controller_requests{exported_namespace=~"{{ .Values.nonProdNamespacePattern }}", ingress=~"{{ $targetIngress }}", path=~"/health", status="503"}[{{ .Values.ingress503ErrorWindowMinutes }}m]) > 0.1) by (ingress, exported_namespace)
```

Comment:
If you set `path="/health"` here - and then `path="/"` on the other rule. - it should separate them nicely.

Reply:
then we can have `status=~"5.*"` on both rules.